backend/internal/schema/hello/routes/routes.go

```diff
@@ -22,7 +23,11 @@ func RouteHelloGroup(app *fiber.App, config *config.Config) {
 	unprotected := app.Group("/hello")
 
 	//Endpoints
-	protected.Get("/world", hello.RetHelloWorld)
-	unprotected.Get("/world", hello.RetHelloWorld)
+	protected.Get("/world", func(c *fiber.Ctx) error {
```

Comment:
if you create a struct with the db as a field, you can simplify this. 
[struct example](https://github.com/GenerateNU/cooked/blob/main/backend/internal/server/handlers/service.go)
[register route example](https://github.com/GenerateNU/cooked/blob/main/backend/internal/server/server.go#L34)